### PR TITLE
Fixed conflict resolution policy crud test

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Fixed conflict resolution policy crud test

### CODE REVIEW

**Review by moderakh — APPROVED** [review]
LGTM. thanks

**Review by simplynaveen20 — APPROVED** [review]
If it become more pain in future as BE keep changing it , we should ignore the '\\\\/a\\\\/b' altogether from the test

**Comment by kushagraThapar** [comment]
> If it become more pain in future as BE keep changing it , we should ignore the '\\/a\\/b' altogether from the test

Good point, we should do that next time.

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/MultiMasterConflictResolutionTest.java [hunk 1]
```diff
@@ -82,8 +82,7 @@ public void conflictResolutionPolicyCRUD() {
             // when (e.StatusCode == HttpStatusCode.BadRequest)
             CosmosException dce = Utils.as(e, CosmosException.class);
             if (dce != null && dce.getStatusCode() == 400) {
-                assertThat(dce.getMessage()).contains("Invalid path '\\\\\\\\\\\\/a\\\\\\\\\\\\/b' for last writer " +
-                    "wins conflict resolution");
+                assertThat(dce.getMessage()).contains("Invalid path '\\\\/a\\\\/b' for last writer wins conflict resolution");
```
Comment (kirankumarkolli) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/MultiMasterConflictResolutionTest.java:85]: Is it a regression? Why isn't it caught before? Where is the source for it? 

Isn't it fragile.

**Pull request merged**